hdate/date.py

```diff
@@ -235,6 +235,23 @@ def omer_day(self):
             return 0
         return omer_day
 
+    @property
+    def daf_yomi(self):
+        """Return a tuple of mesechta and daf."""
+        days_since_start_cycle_11 = (self.gdate - htables.DAF_YOMI_CYCLE_11_START).days
+        daf_number = days_since_start_cycle_11 % (htables.DAF_YOMI_TOTAL_PAGES)
+        for mesechta in htables.DAF_YOMI_MESECHTOS:
+            if daf_number >= mesechta["pages"]:
+                daf_number -= mesechta["pages"]
+            else:
+                break
+        if self.hebrew:
+            heb_number = hebrew_number(daf_number + 2)
+            heb_number = heb_number.replace("'", "").replace('"', "")
+            return mesechta["heb_name"], heb_number
+        else:
+            return mesechta["en_name"], daf_number + 2
```

Comment:
Maybe return daf number as a string so we don't have a discrepancy on the type between English and Hebrew.

Also no need for else as we already returned. A nicer way might be:

    if self.hebrew:
        daf =
        mesechta =
    else:
        ...
    return mesechta, daf

Reply:
Fixed in a6d7f7bce7455c7c608e35e2ebebf52238082df3